app/src/main/java/com/example/androidthings/imageclassifier/ImageClassifierActivity.java

```diff
@@ -80,6 +83,20 @@ protected void onCreate(final Bundle savedInstanceState) {
         mResultViews[0] = (TextView) findViewById(R.id.result1);
         mResultViews[1] = (TextView) findViewById(R.id.result2);
         mResultViews[2] = (TextView) findViewById(R.id.result3);
+        mButton = (android.widget.Button) findViewById(R.id.takePhoto);
```

Comment:
The button isn't necessary here, you can just use `findViewById(...).setOnClickListener(...)`